package.json

```diff
@@ -12,10 +12,12 @@
     "lint": "ng lint",
     "e2e": "ng e2e",
     "build-prod": "node version.js && bash ./scripts/ng.sh build --prod --aot --optimization=false",
+    "build-prod:stats": "npm run build-prod -- --statsJson=true",
     "serve:prod": "http-server dist/ngx-amrs/ -P https://ngx.ampath.or.ke",
     "prebuild": "node version.js",
     "prestart": "node version.js",
-    "prettier:check": "prettier --check src/app"
+    "prettier:check": "prettier --check src/app",
+    "stats": "webpack-bundle-analyzer dist/ngx-amrs/stats.json"
```

Comment:
What if you combine the two commands. Just thinking aloud 🤔
`"stats": "npm run build-prod -- --statsJson=true && webpack-bundle-analyzer dist/ngx-amrs/stats.json"`

Reply:
Sounds good, updated @corneliouzbett